loader_wsock32_proxy/CMakeLists.txt

```diff
@@ -14,7 +14,7 @@ target_link_libraries(loader_wsock32_proxy PRIVATE
                       ${CMAKE_CURRENT_SOURCE_DIR}/include/MinHook.x64.lib
                       mswsock.lib
                       ws2_32.lib
-                      Shlwapi.lib
+                      shlwapi.lib
```

Comment:
Isn't `ShLwApi` the correct casing for this one (note that this differs from the header, `Shlwapi`?

Reply:
this is correct, online information on this has appeared inconcistent.

The correct name as part of the SDK is `ShLwApi.Lib`